src/components/MatchesTable/MatchesTable.tsx

```diff
@@ -174,7 +200,19 @@ export const MatchesTable: FC<MatchesTableProps> = ({
         <MatchLinkCell tag={match.match_id} value={match.duration} />
       ),
     },
-  ];
+    {
+      name: "Mode",
+      selector: (match) => {
+        return match.offensive
+          ? match.attacker
+            ? match.side
+              ? `${match.side} Offensive` : "Offensive"
+            : match.enemy ? `${match.enemy} Offensive` : "Offensive"
+          : "Warfare";
+      },
+      sortable: true,
+    },
+  ].filter(Boolean);
```

Comment:
What's that line supposed to do?

Reply:
It filters out the result column if `null` was returned, i.e. when clanId is undefined, e.g. on the matches page, rather than the clans/[cid] page.

Since the "result" column only makes sense on a clans/[cid] match archive table, but not on the matches/ match archive table.